platformio.ini

```diff
@@ -21,6 +22,7 @@ test_filter =
 
 [env:native]
 platform = native
+build_flags = -std=c++11
```

Comment:
Because the code has already used C++11 features (namely, constructor delegation), let's make it explicit (also makes some nasty warnings go away).